packages/ocaml-option-address-sanitizer/ocaml-option-address-sanitizer.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "Set OCaml to be compiled with address sanitizer"
+description: """
+This configuration package enables memory leak sanitzation using
+the -fsanitize=leak gcc and clang option. Make sure to set
+ASAN_OPTIONS=detect_leaks=0,exitcode=0 while compiling and to
+unset it before running compiled binaries.
```

Comment:
Maybe this should a post-message to make sure the users of this package see it
```suggestion
the -fsanitize=leak gcc and clang option.
"""
post-messages: """
Make sure to set ASAN_OPTIONS=detect_leaks=0,exitcode=0
while compiling and to unset it before running compiled binaries.
```

Reply:
That's a great idea.